stories/theme/ag/small/MobileMapListing.css

```diff
@@ -0,0 +1,121 @@
+.ListingCell {
+  border-radius: 5px;
+  border: 1px solid lightgrey;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.ListingCell .RatingBar_Items {
+  display: flex;
+  flex-direction: row;
+}
+
+.MobileMapListing {
+  width: 280px;
+  height: 200px;
+}
+
+.MobileMapListing_Top {
+  width: 100%;
+}
+
+.MobileMapListing_Bottom {
+  display: flex;
+  width: 100%;
+  justify-content: space-between;
+  padding: 8px 0 11px;
+  height: 80px;
+}
+
+.MobileMapListing_CTAs {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  padding-right: 11px;
+  margin-top: 5px;
+}
+
+.MobileMapListing_CtaButton {
+  padding-top: 5px;
+  padding-bottom: 7px;
+  width: 121px;
+  height: 25px;
+  border-radius: 16px;
+  background-color: #d32526;
+  box-shadow: 0 2px 2px 0 rgba(0, 0, 0, 0.15);
+  font-family: ProximaNova;
+  color: #ffffff;
+  font-size: 11px;
+}
+
+.MobileMapListing .Button {
+  outline: none;
+}
+
+.MobileMapListing_FavoriteButton {
```

Comment:
You shouldn't need to do this. This would be better IMO:
```
.MobileMapListing .FavoriteButton {
```
that way you don't have to add a classname to everything. Things like FavoriteButton should already have one.

Reply:
We thought of that. We actually tried, but there is no `FavoriteButton` component, so to do that we'd have to modify the `themed` call at the top to something like:
```js
@themed(/^(MobileMapListing|FavoriteButton)/, { pure: true })
```

Would you rather us modify the call to `themed` or leave it as is?